src/options/views/edit/index.vue

```diff
@@ -163,6 +192,9 @@ export default {
   created() {
     this.script = this.initial;
     document.addEventListener('keydown', this.switchPanel);
+    if (options.get('editorWindow') && global.history.length === 1) {
+      browser.windows?.getCurrent({ populate: true }, setupSavePosition);
```

Comment:
`browser.windows.getCurrent` should return a `Promise`.

Ref: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/windows/getCurrent

Reply:
Sorry, just woke up...